sources/packages/forms/src/form-definitions/sfaa2025-26-ft.json

```diff
@@ -3017,6 +3017,291 @@
                 "eq": "dependant"
               },
               "lockKey": true
+            },
+            {
+              "label": "Parent Not a BC resident panel",
+              "title": "Parent Not a BC resident panel",
+              "collapsible": false,
+              "hideLabel": true,
+              "key": "parentNotABcResidentPanel1",
```

Comment:
The panel is the only component that should receive the logic to be displayed.
Please add the below logic to the panel and remove it from every component inside the panel.

![image](https://github.com/user-attachments/assets/4e25b480-25b8-492a-9988-088aad4821e8)